src/data/actions/emailTemplate.js

```diff
@@ -49,6 +69,9 @@ const fetchEmailTemplates = options => (
     return EcommerceApiService.fetchEmailTemplate(options)
       .then((response) => {
         dispatch(emailTemplateSuccess(response.data));
+        if (!options.active) {
+          dispatch(allEmailTemplateSuccess(response.data.results));
```

Comment:
I don't think we need to dispatch this action. Just fetch templates from backend and in reducer store them under `assign`, `remind` and `revoke` categories like I mentioned there

Reply:
I have mentioned the reason of the field in upper comment. Please let me know if you think otherwise.

Reply:
I still think we don't need this action. Please see my above comment.